lib/double_entry/reporting/aggregate.rb

```diff
@@ -25,7 +25,7 @@ def amount(force_recalculation = false)
         end
       end
 
-      def formatted_amount(value = amount())
+      def formatted_amount(value = amount)
```

Comment:
the interpreter believes there to be one, i think. But im also a bit unsure as to why it does